projects/saturn/build.gradle

```diff
@@ -69,6 +69,11 @@ dependencies {
     implementation "org.postgresql:postgresql:${postgresqlVersion}"
     implementation "com.zaxxer:HikariCP:5.1.0"
 
+    implementation("org.json:json:20240303")
+
+    // LLM is customer specific. By default the jar will be a demo implementation, overwrite it with customer specific implementation
+    implementation files("libs/nl-hyve-fairspace-demo-v2.4.jar")
```

Comment:
To be honest, I am still not happy with the approach :( We are adding not only the dependency version, but artifact itself too. So, it is pretty tough to change it, too tight coupling. Also, let me know if I am stupid, I am not getting how we can conditionally AskAI switch on and off.